x-pack/plugins/security_solution/public/plugin.tsx

```diff
@@ -355,11 +364,17 @@ export class Plugin implements IPlugin<PluginSetup, PluginStart, SetupPlugins, S
     return {};
   }
 
-  private async downloadAssets() {
-    const [{ renderApp }, { composeLibs }] = await Promise.all([
-      import('./app'),
-      import('./common/lib/compose/kibana_compose'),
-    ]);
+  private async downloadAssets(): Promise<{
```

Comment:
`downloadAssets` `downloadSubPlugins` and `buildStore` are all needed if and only if a user has interacted with on of the registered apps. They all pull their dynamic dependencies from `x-pack/plugins/security_solution/public/lazy_plugin_dependencies.tsx` now. 

We could refactor the code here for clarity but I tried to keep my changes as unobtrusive as possible as the 7.10 feature freeze is nearly here.